modules/test/playwright/pages/object-web/ModelBuilderPage.ts

```diff
@@ -344,6 +351,13 @@ export class ModelBuilderPage {
 			.filter({hasText: objectFolderLabel});
 	};
 
+	getObjectRelationshipDeletionNotAllowedContent = (
+		objectDefinitionName: string,
+		objectFieldName: string
+	) => {
+		return `The object field "${objectFieldName}" cannot be deleted because it is the only custom object field of the published object definition "${objectDefinitionName}". Add at least one object field to the object definition.`;
+	};
+
```

Comment:
What do you think of renaming this method to `getObjectRelationshipFieldDeletionNotAllowedContent` or something like that? Since there could be other reasons for a relationship deletion not to be allowed, with different contents and messages in their warning modals, I think it would be good to make it explicit in here that this method is related to this specific case, where the relationship is also a field.